commons/src/test/java/org/sonarsource/analyzer/commons/RuleMetadataLoaderTest.java

```diff
@@ -530,20 +531,43 @@ class TestRule {
     RulesDefinition.Rule rule = context.repository(RULE_REPOSITORY_KEY).rule("S2092");
     assertThat(rule.type()).isEqualTo(RuleType.SECURITY_HOTSPOT);
     assertThat(rule.securityStandards())
-      .containsExactlyInAnyOrder("cwe:311", "cwe:315", "cwe:614", "owaspTop10:a2", "owaspTop10:a3", "owaspTop10-2021:a4", "owaspTop10-2021:a5");
+      .containsExactlyInAnyOrder("cwe:311", "cwe:315", "cwe:614",
+        "owaspTop10:a2", "owaspTop10:a3",
+        "owaspTop10-2021:a4", "owaspTop10-2021:a5");
+  }
+
+  @Test
+  public void test_security_standards_on_13_3() {
+    Set<String> securityStandards = getSecurityStandards(SONAR_RUNTIME_13_3);
+    assertThat(securityStandards).containsExactlyInAnyOrder(
+      "cwe:311", "cwe:315", "cwe:614",
+      "masvs-1:MSTG-STORAGE-14", "masvs-2:MASVS-STORAGE-15",
+      "owaspTop10:a2", "owaspTop10:a3",
+      "owaspTop10-2021:a4", "owaspTop10-2021:a5",
+      "owaspTop10-2025:a6", "owaspTop10-2025:a7",
+      "pciDss-3.2:1.1.1", "pciDss-3.2:1.1.2",
+      "owaspAsvs-4.0:2.1.1", "owaspAsvs-4.0:2.1.2",
+      "owaspAsvs-5:2.1.3", "owaspAsvs-5:2.1.4",
+      "stig-ASD_V5R3:V-222612",
+      "stig-ASD_V6:V-222613",
+      "owaspMobileTop10-2024:m3", "owaspMobileTop10-2024:m4",
+      "owaspLlmTop10-2025:llm01", "owaspLlmTop10-2025:llm10"
```

Comment:
Having three, two, or just one value on a line makes it harder to read.

Reply:
I prefer the second option. While grouping standards by line can be helpful, it is not clear what would happen if we have six OWASP mobile standards, as we could exceed the character limit for that line.